gadd/src/main/java/textDictionary/Run.java

```diff
@@ -0,0 +1,15 @@
+package textDictionary;
+
+import java.util.HashMap;
+import java.util.Set;
+
+public class Run {
+    public static void main(String[] args) {
+        CreateDictionary cd = new CreateDictionary("/home/gadd/example_text.txt");
+        HashMap dict = cd.getResult();
+        Set dictKeys = dict.keySet();
+        for (Object key : dictKeys) {
```

Comment:
it is better to iterate by Entity. f.e. :
        Map<String, Integer> zu = new HashMap<>();
       for (Map.Entry<String, Integer> item : zu.entrySet()) {
            System.out.println(item.getKey()+" "+item.getValue());
        }